src/main/java/hudson/plugins/spotinst/api/SpotinstApi.java

```diff
@@ -313,6 +317,75 @@ public static Boolean azureVmDetach(String groupId, String vmId, String accountI
     }
     //endregion
 
+    //Redis
+    //TODO Liron - change name to getLock
+    public static <T> T getRedisValue(String groupId, String accountId) throws ApiException {
+        T retVal      = null;
+
+        Map<String, String>    headers     = buildHeaders();
+
+        Map<String, String>    queryParams = buildQueryParams(accountId);
+
+        RestResponse response =
+                RestClient.sendGet(SPOTINST_API_HOST + "/aws/ec2/group/" + groupId + "/jenkinsPlugin", headers, queryParams);
```

Comment:
Add the suffix "lock" to the url (and change the optimizer)

Reply:
finished in plugin, optimizer and core service remains